<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="createButton">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>340</y>
      <width>271</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Створити об'єкт</string>
    </property>
   </widget>
   <widget class="QLabel" name="new_2">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>240</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Ім'я</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="nameLineEdit">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>270</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="surnameLineEdit">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>270</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="addressLineEdit">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>270</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="information">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>240</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Прізвище</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>250</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Адреса</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionCreate"/>
    <addaction name="actionExit"/>
    <addaction name="actionShow"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action">
   <property name="text">
    <string>Створити об'єкт</string>
   </property>
  </action>
  <action name="action_2">
   <property name="text">
    <string>Відобразити об'єкт</string>
   </property>
  </action>
  <action name="action_3">
   <property name="text">
    <string>Вихід</string>
   </property>
  </action>
  <action name="actionCreate">
   <property name="text">
    <string>Створити об'єкт</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Вихід</string>
   </property>
  </action>
  <action name="actionShow">
   <property name="text">
    <string>Відобразити об'єкт</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
